functions/src/tasks/index.js

```diff
@@ -1,5 +1,5 @@
 const admin = require("firebase-admin");
-const serviceAccount = require("../../credentials.json");
+const serviceAccount = require("../../cred.json");
```

Comment:
Please name your file the same as mine, "credentials.json". While you do create your own file, we share the same relative import, therefore if I merge this, it will break the import on my local.

Reply:
done